.github/workflows/release-tagger.yaml

```
@@ -24,12 +24,6 @@ jobs:
           # Extract vX from vX.Y.Z
           NEW_TAG=$(echo "${{ github.event.release.tag_name }}" | cut -d'.' -f1)
 
-          # Check if tag already exists and delete if it does
-          if git show-ref --tags $NEW_TAG; then
-              git tag -d $NEW_TAG
-              git push --delete origin $NEW_TAG
-          fi
-
           # Create new tag
-          git tag $NEW_TAG
-          git push origin refs/tags/$NEW_TAG
+          git tag -f $NEW_TAG -m "Forcing tag to match release"
+          git push origin -f --tags
```

Instead of force pushing all tags, it may be better to only push this particular tag.

```suggestion
          git push origin -f $NEW_TAG
```